Generate.ps1

```diff
@@ -183,7 +183,7 @@ Remove-Item ./azure-dataplane-tests/src/main/java/module-info.java -Force | Out-
 
 # Azure
 $job = @(
-    "$AZURE_ARGUMENTS --input-file=$SWAGGER_PATH/paging.json --namespace=fixtures.paging --payload-flattening-threshold=1 --generate-sync-async-clients",
+    "$AZURE_ARGUMENTS --input-file=$SWAGGER_PATH/paging.json --namespace=fixtures.paging --payload-flattening-threshold=1 --generate-sync-async-clients --use-iterable",
```

Comment:
this should fail in current code
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4859703&view=logs&j=81a719ea-44f7-5728-14cb-b369713e6844&t=5fa2c1cd-7130-5a2f-6bd7-6d27b879d56b&l=414